src/bot/database/entities/streamer.py

```diff
@@ -14,8 +14,8 @@ class Streamer(Base):
     id = Column(Integer, primary_key=True, autoincrement=True, unique=True)
     twitch_id = Column(Integer)
     twitter_id = Column(Integer)
-    name = Column(String)
     category_id = Column(ForeignKey("categories.id"))
+    name = Column(String)
```

Comment:
alteração não necessaria

Reply:
This change is necessary so that the integers come before the string fields so that the database doesn't lose performance due to silly things.
Another reason this is done is because the diagram has been redone so we can work from scratch with these tables.